third_party/efr32_sdk/efr32_sdk.gni

```diff
@@ -182,7 +185,9 @@ template("efr32_sdk") {
     } else if (efr32_family == "efr32mg21") {
       _include_dirs += [
         "${efr32_sdk_root}/hardware/kit/EFR32MG21_${efr32_board}/config",
-        "${efr32_sdk_root}/platform/Device/SiliconLabs/EFR32MG21/Include",
+        "${efr32_sdk_root}/hardware/kit/MGM21_${efr32_board}/config",
```

Comment:
Including two different kit configs might be problematic here. I dont know the MGM21 module and am I not sure what are the difference but there must be some issues with the pin defines since you mention issues with leds and LCD